code/swb_base/Weapon.cs

```diff
@@ -301,7 +304,26 @@ void CreateModels()
 			ViewModelHandler = viewModelGO.Components.Create<ViewModelHandler>();
 			ViewModelHandler.Weapon = this;
 			ViewModelHandler.ViewModelRenderer = ViewModelRenderer;
-			ViewModelHandler.Camera = Owner.ViewModelCamera;
+			var viewModelCamera = Owner.ViewModelCamera;
+			if ( Owner.ViewModelCamera is null )
+			{
+				var viewModelCameraGameObject = Owner.GameObject.Scene.CreateObject( true );
```

Comment:
This should do the same: ` Owner.GameObject.Scene.CreateObject( true );` => `new GameObject();`